Scripts/dataGenerator.R

```diff
@@ -0,0 +1,21 @@
+library(CRCSim)
+
+unevenCaptures_Strata <- runCRC(100, 1,
+                                p_captures = c(0.9, 0.2, 0.05, 0.02, 0.1, 0.4),
+                                p_strata = c(.8, .05, 0.05, 0.15, 0.01))
+write.csv(unevenCaptures_Strata, "../Data/unevenCaptures_Strata.csv", row.names = FALSE)
```

Comment:
I'm not a huge fan of these paths for re-usability. Can we set them as a variable that we pass in?

Reply:
I think top level will have Scripts/ Figures/ CRCSim/ and app.r + the manuscript which feels like a good setup for journal submission with intuitive pathing.

The reason why this file is extremely explicit is to aid people who may be checking this in the future (who aren't programmers) else it's easily functionalized

Reply:
Resolving since this goes to write.csv and the relative path is part of the containerized repo.